### ISSUE TITLE
Cannot create New entry, after clicking Update (and seeing that no entry is actually correct)

### ISSUE THREAD

**Issue by stdedos** [keepassxreboot/keepassxc-browser#2216]
## Expected Behavior

Even after clicking "Update" to update an existing entry, the user might want again to create a new entry (because the "correct" entry is not coming up)

## Current Behavior

Clicking "Update" is final.

## Steps to Reproduce (for bugs)

1.
2.
3.
4.

## Debug info

KeePassXC - 2.7.8
KeePassXC-Browser - 1.9.0.3
Operating system: Win64
Browser: Chrome/Chromium 117.0.0.0

**Comment by varjolintu** [comment]
I'm not 100% sure what you mean here. Could you clarify this a little bit?

**Comment by stdedos** [comment]
Clicking update here

![image](https://github.com/keepassxreboot/keepassxc-browser/assets/133706/d571f2f3-84ef-4e04-a800-e5dc153c42d3)

and the "for some reason" realizing that there is no suitable account to _update_, means you are "loosing the possibility" to save the credentials-in-question (either you don't save them, and you log out / log in again to trigger this - or you "update" a wrong entry, and you loose that entry)

**Comment by varjolintu** [comment]
If the Update is not triggered/activated, you won't lose the account if you click the New button, right?

**Comment by stdedos** [comment]
AFAIS New button disappears after clicking Update. In my case, I had multiple entries to consider, and none of them were the correct one.

**Comment by varjolintu** [comment]
You can cancel the update, and you have still both options available.

**Comment by stdedos** [comment]
> You can cancel the update, and you have still both options available.

I don't see where you see that 😕 

On the current test system, I have two test DBs.

1. I selected one.
2. KeepassXC asked me "do I want to update the entry".
3. I said "Cancel".

I saw this

![image](https://github.com/keepassxreboot/keepassxc-browser/assets/133706/d673bf5d-9e28-4cbd-8f0c-e06f2fbe2e3f)

**Comment by droidmonkey** [comment]
Seems like a logical result to the question asked and the answer given.

**Comment by stdedos** [comment]
> Seems like a logical result

Yes. Maybe. idk.

My question/request is that I can change my mind in the Update operation, if it is not yet committed, i.e.:

1. There is only one credential and I haven't clicked Yes on the "do I want to update the entry" in KeepassXC, or
2. There are multiple entries, from which I should choose which one to update

**Comment by stdedos** [comment]
> Clicking update here
> 
> ![image](https://private-user-images.githubusercontent.com/133706/336467901-d571f2f3-84ef-4e04-a800-e5dc153c42d3.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTc4NTE2NDUsIm5iZiI6MTcxNzg1MTM0NSwicGF0aCI6Ii8xMzM3MDYvMzM2NDY3OTAxLWQ1NzFmMmYzLTg0ZWYtNGUwNC1hODAwLWU1ZGMxNTNjNDJkMy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNjA4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDYwOFQxMjU1NDVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02NGI2MzY5YTkyNTRiNmMwZjkyMGNiMzQwZmU5OTY4NWEwZWZiYzA5YzFhOWFlZWFiOWRlYmJkNjNhMjZjODBjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9._6W_8Im4SnpaWXY_L_FkEu1pziMNs5kvfNCdJPwBBkc)
> 
> and the "for some reason" realizing that there is no suitable account to _update_, means you are "loosing the possibility" to save the credentials-in-question (either you don't save them, and you log out / log in again to trigger this - or you "update" a wrong entry, and you loose that entry)

Here

![image](https://github.com/keepassxreboot/keepassxc-browser/assets/133706/99939c43-b885-4cb7-b794-119c1cb1e061)

There is no `New` button any more

**Comment by varjolintu** [comment]
But there's the "Dismiss" button and it wil return to the step where both New and Update buttons are shown again.

**Comment by stdedos** [comment]
`Dismiss` is the same button as in the first version of the pop-up. There, it means "No, I don't want to save that entry".

Neither the color, nor the text is changed. Why would I guess that it is a different action (i.e., back)?
Why would I need a back/forward mechanism, when I could've just pressed `New` again?

**Comment by varjolintu** [comment]
Dismiss lets you decide again. Cancel would cancel the whole process.

**Comment by varjolintu** [comment]
Maybe the button text could be changed to "Back", yes. I still fail to see the original issue here.

**Comment by stdedos** [comment]
I just tested it: Dismiss here

![image](https://github.com/keepassxreboot/keepassxc-browser/assets/133706/99939c43-b885-4cb7-b794-119c1cb1e061)

_does_ get you back to this 

![image](https://github.com/keepassxreboot/keepassxc-browser/assets/133706/d571f2f3-84ef-4e04-a800-e5dc153c42d3)

tl;dr it's a UX issue.

Kindly consider renaming that "Dismiss" button as back on the second state - or any other way you deem appropriate to inform/differentiate the first and the second "Dismiss" buttons.

**Comment by varjolintu** [comment]
Created a PR where buttons are renamed.
